src/tests/decider-distro-tests-rs/tests/test_subnet.rs

```diff
@@ -6,6 +6,13 @@ pub mod utils;
 
 use fluence_spell_dtos::trigger_config::TriggerConfig;
 use serde_json::json;
+use utils::chain::{filter_logs, LogsReq};
+use utils::control::{update_config, update_decider_script_for_tests, wait_decider_stopped};
+use utils::deal::{get_failed_deals, get_joined_deals};
+use utils::default::{default_receipt, DEAL_IDS};
+use utils::distro::make_distro_with_api_and_config;
+use utils::setup::setup_nox;
+use utils::subnet::{get_txs, get_txs_statuses};
 use utils::test_rpc_server::run_test_server;
 use utils::TestApp;
 use utils::*;
```

Comment:
Do you need that after all the imports above?

Reply:
Yeah, there are still small helpers that are just utils and not sorted into anything more sensible than "utils"